my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.post-title {
+  color: #37474f;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.post-url {
+  font-size: 12px;
+  margin-left: 8px;
+  color: #0277bd;
+  line-height: 24px;
+}
+
+.post-points {
+  font-size: 10px;
+  margin: 0 8px 0 0;
+}
+
+.post-second-line-item {
+  font-size: 10px;
+  margin: 0 8px 0 0;
+  color: #0277bd;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.btn {
+  box-shadow: none;
+  font-family: "Lato", sans-serif;
+  border: none;
+  background: #f4511e;
+  border-radius: 4px;
+  padding: 0 32px;
+  display: inline-block;
+  margin: 16px 0 40px 32px;
+  height: 40px;
+  line-height: 40px;
+  color: #ffffff;
+  text-transform: uppercase;
+  outline: none;
+  -webkit-transition: all 0.3s;
+  transition: all 0.3s;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.App {
+  text-align: left;
+}
+
+.App-logo {
```

Comment:
I suggest to be more consistent in naming your classes. For example, `.post-xx` are lower case but `.App-logo` is upper case.